website/src/components/Schema/Regex.tsx

```diff
@@ -20,33 +20,33 @@
 
 /** @jsx jsx */
 import { jsx, css } from '@emotion/core';
-import React from 'react';
-import styles from './styles.module.css';
 import HighlightedRegex from './HighlightedRegex';
-import get from 'lodash/get';
-import { ChangeType } from '.';
+import isEmpty from 'lodash/isEmpty';
 
-export const RegexExamples = ({ regex, examples }) => (
-  <div>
-    <br />
-    <div>Examples:</div>
+export const RegexExamples = ({ regex, examples }) => {
+  if (isEmpty(examples)) return null;
```

Comment:
could also just bundle this check in the parent where the other checks are happening eh? I think that's a bit easier to follow cause we don't have to drop down into this level to figure out that there's another gate in here...

What do you think?